src/panels/config/users/dialog-user-detail.ts

```diff
@@ -192,11 +203,16 @@ class DialogUserDetail extends LitElement {
     this._isAdmin = ev.target.checked;
   }
 
+  private async _activeChanged(ev): Promise<void> {
+    this._isActive = ev.target.checked;
```

Comment:
Let's add a confirmation if you are deactivating a user

Reply:
Do we really need that? It is a non-destructive process, carries low risk and is clearly visible. We do not have a confirmation for the "Administrator" switch either, which carries a greater impact IMHO since it gives far reaching access.

If you insist on the switch, should it come up directly when the switch value is changed or only once the users presses the "Update" button of the dialog?

Reply:
I guess we have the update button, so the action is not immediately... So it is not needed maybe